src/components/VmDetails/cards/UtilizationCard/DiskCharts.js

```diff
@@ -43,18 +72,20 @@ const DiskCharts = ({ vm, isRunning, id, ...props }) => {
   })
 
   const { unit, value: disks } = convertValueMap('B', { actualSize, provisionedSize })
-  const used = round(disks.actualSize, 1)
   const available = round(disks.provisionedSize - disks.actualSize, 1)
   const total = round(disks.provisionedSize, 1)
 
   return (
     <UtilizationCard className={style['chart-card']} id={id}>
       <CardTitle>Disk <span style={{ fontSize: '55%', verticalAlign: 'super' }}>(storage allocations)</span></CardTitle>
       <CardBody>
-        { !hasDisks &&
-          <NoLiveData id={`${id}-no-live-data`} message='This VM has no attached disks.' />
+        { !diskDetails && isRunning &&
+          <NoLiveData id={`${id}-no-live-data`} message='It seems that no guest agent is configurated on VM.' />
```

Comment:
This reminds me that Utilization Card needs to be localized.  Perhaps you can start with this message.

Reply:
It is better to do in separate PR, all at once: https://github.com/oVirt/ovirt-web-ui/issues/879